### PR TITLE
[TAS-743]💄 Add title to collector table

### PR DESCRIPTION
**Pull request by AuroraHuang22**
截圖 2023-12-04 下午5 09 25

### CODE REVIEW

File: src/components/NFTPage/CollectorList/index.vue [hunk 1]
```diff
@@ -26,37 +55,39 @@
...
                 <tbody class="w-full">
+                  <tr class="w-full border-b-shade-gray border-b-[1px]">
+                    <td class="my-[12px]">
+                      <Label
+                        class="w-min text-[12px] font-500"
+                        :text="$t('nft_details_page_title_collector')"
+                        valign="middle"
+                        content-class="whitespace-nowrap text-dark-gray"
+                      >
+                      </Label>
```
Comment (nwingt) [inline src/components/NFTPage/CollectorList/index.vue:70]:
```suggestion
                      />
```

File: src/components/NFTPage/CollectorList/index.vue [hunk 2]
```diff
@@ -26,37 +55,39 @@
...
-              />
-            </div>
-            <hr class="w-full border-shade-gray" />
-          </template>
           <template #content>
             <div class="min-w-[310px]">
               <table class="w-full">
                 <tbody class="w-full">
+                  <tr class="w-full border-b-shade-gray border-b-[1px]">
+                    <td class="my-[12px]">
```
Comment (nwingt) [inline src/components/NFTPage/CollectorList/index.vue:63]: Can we use `v-for` for these 3 `<td />`, also suggest to add a vertical padding
```suggestion
                    <td class="py-[12px]">
```

File: src/components/NFTPage/CollectorList/index.vue [hunk 3]
```diff
@@ -26,37 +55,39 @@
...
-                class="w-min font-600"
-                :text="$t('nft_details_page_label_owning')"
-                preset="h5"
-                valign="middle"
-                content-class="whitespace-nowrap text-like-green "
-                prepend-class="text-like-green"
-              />
-            </div>
-            <hr class="w-full border-shade-gray" />
-          </template>
```
Comment (nwingt) [inline src/components/NFTPage/CollectorList/index.vue:55]: Why remove this?

File: src/components/NFTPage/CollectorList/index.vue [hunk 4]
```diff
@@ -13,6 +13,35 @@
         <div :class="contentPaddingClass">
           <table class="w-full">
             <tbody class="w-full">
+              <tr class="w-full border-b-shade-gray border-b-[1px]">
+                <td class="my-[12px]">
```
Comment (nwingt) [inline src/components/NFTPage/CollectorList/index.vue:17]: Same here

File: src/components/NFTPage/CollectorList/index.vue (hunk 3 above)
Comment (AuroraHuang22) [reply src/components/NFTPage/CollectorList/index.vue:55]: I wanted to hide the header like the history modal, but let's align the UI maybe later ~

File: src/components/NFTPage/CollectorList/index.vue [hunk 5]
```diff
@@ -13,6 +13,35 @@
...
+                  />
+                </td>
+                <td>
+                  <Label
+                    class="w-min text-dark-gray text-[12px] font-500"
+                    :text="$t('nft_details_page_label_owning')"
+                    valign="middle"
+                    content-class="whitespace-nowrap"
+                  />
+                </td>
```
Comment (nwingt) [inline src/components/NFTPage/CollectorList/index.vue:43]: v-for?

File: src/components/NFTPage/CollectorList/index.vue [hunk 6]
```diff
@@ -57,6 +86,34 @@
             <div class="min-w-[310px]">
               <table class="w-full">
                 <tbody class="w-full">
+                  <tr class="w-full border-b-shade-gray border-b-[1px]">
+                    <td class="py-[12px]">
```
Comment (nwingt) [inline src/components/NFTPage/CollectorList/index.vue:90]: v-for?

**Comment by AuroraHuang22** [comment]
[♻️ Refactor title row using v-for](https://github.com/likecoin/liker-land/pull/1448/commits/6c2b55a00d4241ca06641012656597d17b54b336)
截圖 2023-12-05 上午11 49 52
截圖 2023-12-05 上午11 50 00

File: src/components/NFTPage/CollectorList/index.vue [hunk 7]
```diff
@@ -157,6 +124,15 @@ export default {
       default: false,
     },
   },
+  data() {
+    return {
+      labelTexts: [
```
Comment (nwingt) [inline src/components/NFTPage/CollectorList/index.vue:129]: can move to computed as the text does not change, and rename to  `tableHeaderItems`?

**Pull request merged**